ClientLib/src/main/java/org/droidplanner/services/android/impl/core/drone/autopilot/apm/ArduPilot.java

```diff
@@ -386,15 +385,15 @@ protected boolean performTakeoff(Bundle data, ICommandListener listener) {
     @Override
     public void onMavLinkMessageReceived(MAVLinkMessage message) {
 
-        if (message.sysid != this.getSysid()) {
+        if ((message.sysid != this.getSysid()) && !isMavLinkMessageException(message)) {
             // Reject Messages that are not for the system id
             return;
         }
 
         int compId = message.compid;
         if (compId != AUTOPILOT_COMPONENT_ID
                 && compId != ARTOO_COMPONENT_ID
-                && compId != TELEMETRY_RADIO_COMPONENT_ID) {
+                && !(message.sysid == SiK_RADIO_FIXED_SYSID && compId == SiK_RADIO_FIXED_COMPID) ){
```

Comment:
@billbonney Here can you use `isMavlinkMessageException(...)` as well since the predicate is the same.